guide/src/config.md

```diff
@@ -101,10 +101,12 @@ A `chains` section includes parameters related to a chain and the full node to w
 
 * __trusting_period__: *(string)* Specify the amount of time to be used as the light client trusting period. It should be significantly less than the unbonding period (e.g. unbonding period = 3 weeks, trusting period = 2 weeks). Default: `14days` (336 hours)
 
-* __trust_threshold__: *(table)* Specify the trust threshold for the light client, ie. the maximum fraction of validators which have changed between two blocks. Default: `{ numerator = '1', denominator = '3' }`, ie. 1/3.
+* __trust_threshold__ <sup>(advanced)</sup>: *(table)* Specify the trust threshold for the light client, ie. the maximum fraction of validators which have changed between two blocks. Default: `{ numerator = '1', denominator = '3' }`, ie. 1/3.
```

Comment:
Cool!

BTW, it is also important to add the warning in the ibc-rs/config.toml file

Reply:
Done 👍